app/livechat/imports/server/rest/visitors.js

```diff
@@ -3,6 +3,7 @@ import { check } from 'meteor/check';
 
 import { API } from '../../../../api';
 import { findVisitorInfo, findVisitedPages, findChatHistory } from '../../../server/api/lib/visitors';
+import { normalizeMessagesForUser } from '/app/utils/server/lib/normalizeMessagesForUser';
```

Comment:
Why did you import this method if you're not using it?

Reply:
removed